landing_page/templates/landing_page/base.html

```diff
@@ -49,15 +46,16 @@ <h1>ReeldIn</h1>
     </div>
     <div id="profile_div">
       <a href="{% url 'recommendations:index' %}">
-        <button id="get_recommendations_btm">
-          Get Recommendations
-        </button>
+        <button id="get_recommendations_btm">Get Recommendations</button>
       </a>
       <a href="{% url 'accounts:login' %}">
         <button id="login_button">Login</button>
       </a>
-      <img id="profile_pic" src="{% static 'landing_page/images/profile.png' %}" />
-      </img>
+      <img
+        id="profile_pic"
+        src="{% static 'landing_page/images/profile.png' %}"
```

Comment:
placeholder for now right?
also feel like the profile picture should be in the static 'accounts/images' folder yeah?